assets/src/components/mapMarkers.tsx

```diff
@@ -257,72 +257,113 @@ export const StationMarker = React.memo(
   }
 )
 
-export const RouteStopMarkers = ({
+/**
+ * @returns a list of stops at unique locations. Where a platform stop is at the exact location of a station, the list will include the station.
+ */
+const uniqueStopsByLocation = (stops: Stop[]) => {
+  const locationToStop: Record<string, Stop> = {}
+  stops.forEach((stop) => {
+    const key = `${stop.lat}_${stop.lon}`
```

Comment:
**question:** I haven't thought about this in great depth, but do you know if there's any possibility that floating point weirdness might lead to stops that should match not matching?

Reply:
Ooh I hadn't considered that! I don't think there should be floating point issues affecting the string equality here, at least nothing jumped out at me in the [toString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference) doc. 

I also suspect if there were an issue that would cause two co-located stops to have slightly different keys, the distance between the center of those stops would be small (<3in based on [this comment](https://github.com/mbta/skate/blob/1f9716979afd3b2abfb8de96e1bec32b291ab5db/assets/tests/factories/geoCoordinate.ts#L11)). If that were to happen, I think the centers would be so close that one marker would effectively be on top of the other and only one would be visible + clickable to the user.

Reply:
Sounds good to me!